### ISSUE TITLE
RTX 4090Ti GPUs, Batch Size Reduction, and Performance Challenges in Depth Estimation

### ISSUE THREAD

**Issue by wonjsu** [wl-zhao/VPD#59]
I used two RTX 4090Ti GPUs and reduced the batch size to 2 due to the limitations of the GPUs. Subsequently, I changed batch normalization to layer normalization, and additionally, I decreased the learning rate by a factor of 1/10. However, the performance did not improve. I would appreciate advice on how to address this issue.
